metadata-ingestion/src/datahub/ingestion/source/qlik_sense/data_classes.py

```diff
@@ -164,8 +164,8 @@ class Chart(BaseModel):
     @root_validator(pre=True)
     def update_values(cls, values: Dict) -> Dict:
         values[Constant.QID] = values[Constant.QINFO][Constant.QID]
-        values["qDimension"] = values["qHyperCube"]["qDimensionInfo"]
-        values["qMeasure"] = values["qHyperCube"]["qMeasureInfo"]
+        values["qDimension"] = values[Constant.HYPERCUBE]["qDimensionInfo"]
```

Comment:
Can we use Constans instead of raw strings like what you did with `HYPERCUBE`?
I would prefer to be consistent in the code.